app/v2/controllers/V2MovementsController.scala

```diff
@@ -533,7 +533,8 @@ class V2MovementsControllerImpl @Inject() (
                  |Reason: ${failureDetails.failureReason}
                  |Message: ${failureDetails.message}""".stripMargin)
 
-            // TODO: Push notification on failure
```

Comment:
@drnaylor-oc  I think we also need to set the status as failed here in db for this use case?